client/src/constants/validation.js

```diff
@@ -210,12 +210,12 @@ export const HiredParticipantSchema = yup.object().noUnknown('Unknown field in f
     then: yup.string().required('Please specify'),
     otherwise: yup.string().nullable().test('is-null', 'Other origin must be null', (v) => v == null || v === ''),
   }),
+  nonHcapOpportunity: yup.boolean().required('Non-Hcap Opportunity is required as true or false'),
   contactedDate: yup.string().required('Date of contact is required').test('is-date', 'Not a valid date in the past', validatePastDateString),
+  hiredDate: yup.string().required('Date offer accepted  is required').test('is-date', 'Not a valid date in the past', validatePastDateString),
```

Comment:
order changed to match other instances of the field list & renamed offerDate to hiredDate